pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/SinksImpl.java

```diff
@@ -232,6 +233,10 @@ public void createSink(SinkConfig sinkConfig, String fileName) throws PulsarAdmi
     @Override
     public CompletableFuture<Void> createSinkAsync(SinkConfig sinkConfig, String fileName) {
         final CompletableFuture<Void> future = new CompletableFuture<>();
+        if (StringUtils.isBlank(sinkConfig.getName())) {
```

Comment:
Don't we need a better story here to validate the parameters? For example, what if tenant or namespace is null?

Reply:
@sijie I have added validation for all of the methods in SinksImpl for sinkname, namespace and tenant.
PTAL